src/components/Editor/SearchBar.js

```diff
@@ -304,6 +313,7 @@ class SearchBar extends Component<Props, State> {
           handleNext={e => this.traverseResults(e, false)}
           handlePrev={e => this.traverseResults(e, true)}
           handleClose={this.closeSearch}
+          ref={c => (this.$input = c)}
```

Comment:
hmm, we've generally kept only json in redux. @wldcordeiro might have a good idea here